barretenberg/cpp/src/barretenberg/ultra_honk/oink_prover.cpp

```diff
@@ -63,6 +63,8 @@ template <IsUltraFlavor Flavor> void OinkProver<Flavor>::prove()
     // Free the commitment key
     proving_key->proving_key.commitment_key = nullptr;
     // #endif
+
+    return transcript->proof_data;
```

Comment:
This returned proof is used in CIVC but it doesn't need to be used elsewhere e.g. in UH where Oink is just the first stage